src/routes/api/userRoute.js

```diff
@@ -129,4 +129,5 @@ const { isImage } = customValidator;
 */
 userRoute.put('/profile-settings', verifyToken, profileUpdateRules(), checkInputDataError, isImage, isManager, isUserVerified, UserController.updateProfile);
 userRoute.get('/view-profile', verifyToken, isUserVerified, UserController.viewProfile);
+userRoute.get('/notification', verifyToken, isUserVerified, UserController.viewNotification);
```

Comment:
@NiyoEric  please add swagger doc for this route.

Reply:
I have added it.